content/courses/tokens-and-nfts/nfts-with-metaplex-core.md

```diff
@@ -0,0 +1,834 @@
+---
+title: Create Solana NFTs With Metaplex Core
+objectives:
+  - Explain NFTs and how they're represented on the Solana network
+  - Explain Collections and how do they differ between the Core Program and the Token Metadata Program
+  - Explain the role of the Metaplex Core program
+  - Create and update a Collection using the Metaplex JS SDK
+  - Create and update Assets using the Metaplex JS SDK
+description:
+  "How to create NFTs in TypeScript with Metaplex Core program and Irys
+  permanent storage service."
+tags:
+  - quickstart
+  - metaplex
+  - typescript
+keywords:
+- typescript
+- npm
+- yarn
+- package
+- tutorial
+- digital assets
+- intro to solana nfts
+- blockchain developer
+- blockchain tutorial
+- web3 developer
+---
+
+### Summary
+
+- **Non-Fungible Tokens (NFTs)** are Digital Assets with 0 decimals, and 
+  a maximum supply of 1
+- **Metadata** attaches additional properties to Assets and Collection. 
+  Metadata includes the token name and a link to an offchain JSON file. 
+  This JSON file contains links to artwork and other media files, any 
+  special traits the NFT has, and more.
+- The [Metaplex Core Program](https://developers.metaplex.com/core) is an 
+  onchain program that sheds the complexity and technical debt of previous 
+  standards and provides a clean and simple core spec for digital assets 
+  using a single account design
+
+### Lesson
+
+Solana Non-Fungible Tokens (NFTs) used to be represented as SPL tokens 
+
+In this lesson, we'll explore how Core Assets are represented and demonstrate 
+how to create and update them using the mpl-core npm module.
+
+Solana Non-Fungible Tokens (NFTs) used to be represented as SPL tokens 
+with an additional metadata account associated with each token mint and 
+created using both the Token Program and the Metaplex Token Metadata Program.
+
+However, with the introduction of the new Metaplex Core Program, NFTs 
+have their own program and standard that leverage a single account design 
+and has a flexible plugin system that that enables developers to natively 
+modify asset behavior and functionality
+
+In this lesson, we'll explore how Core Assets are represented and demonstrate 
+how to create and update them using the `mpl-core` npm module.
+
+#### NFTs on Solana
+
+All NFTs must meet the following characteristics:
+
+1. Have 0 decimals, so it cannot be divided into parts.
+2. Have a supply of 1, so only 1 of these tokens exists.
+3. Not have a mint authority to ensure that the supply never changes.
+4. Have an associated **metadata** to store things like a name, images, etc.
+
+While all this properties were previously achievable with a combination of 
+the SPL Token Program and the Metaplex Token Metadata Program, it came with 
+a big overhead and lots of inefficiencies for a market as big as NFTs and Digital 
+Asset in general.
+
+This is exaclty why the **Metaplex Core Program** was created!
```

Comment:
This again feels like you are stating THIS is what NFTs are, but we are not doing that. 

This feels like we are teaching more about old NFT's and stating we are actually not doing this.

Feel there should be a differences area like above the above comment to.